### PR TITLE
feat(membership): added model test to membership model

### PR DESCRIPTION
**Pull request by Her0elt**
## Made tests for the membership model

related to #154 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [x] Build (`make PR`) was run locally without failures

### CODE REVIEW

File: app/group/factories/membership_factory.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class Membershipfactory(DjangoModelFactory):
+    """Factory that creates a generic membership"""
+
+    class Meta:
+        model = Membership
+
+    user = factory.SubFactory(UserFactory)
+    group = factory.SubFactory(GroupFactory)
+    membership_type = factory.Iterator(MembershipType)
+    expiration_date = timezone.now()
```
Comment (Zenjjim) [inline app/group/factories/membership_factory.py:21]: Denne skal vel ikke bli satt til now()?

File: app/group/models/membership.py [hunk 2]
```diff
@@ -58,11 +58,11 @@ def __str__(self):
     def swap_leader(self):
```
Comment (Zenjjim) [inline app/group/models/membership.py:58]: burde være swap_board()

**Review by Zenjjim — CHANGES_REQUESTED** [review]

File: app/group/tests/test_membership_model.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+@pytest.fixture()
+def group():
+    return GroupFactory()
+
+
+@pytest.mark.django_db
+def test_swap_leader_changes_leader(group):
+    """Tests that swap leader changes the leader of the group,
+     when there is a leader and a member"""
```
Comment (Eirsteir) [inline app/group/tests/test_membership_model.py:29]: Multiline docstrings should according to PEP8 be on a new line, ie:
```
def amazing():
"""
This function
is amazing.
"""
```

File: app/group/tests/test_membership_model.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+     the function raises a ValidationError"""
+    with pytest.raises(ValidationError):
+        membership_leader.swap_board(MembershipType.LEADER)
+
+
+@pytest.mark.django_db
+def test_swap_board_raises_does_not_exist(membership):
+    """Tests that if a member tries to use swap_leader,
+     when there is no leader in the group it raises a DoesNotExist error"""
+    with pytest.raises(Membership.DoesNotExist):
```
Comment (Eirsteir) [inline app/group/tests/test_membership_model.py:53]: Is it necessary to test for this? I feel like this only tests Django logic

**Review by Eirsteir — CHANGES_REQUESTED** [review]
Very nice!  Nothing major to comment on 😉

**Review by Zenjjim — APPROVED** [review]

**Review by Eirsteir — APPROVED** [review]
🥳 🥳

**Pull request merged**